Description: Fix PackageKit package name matches for xz and rpm files.
Author: Jordi Mallach <jordi@debian.org>

--- a/data/packages.match
+++ b/data/packages.match
@@ -16,16 +16,16 @@
 p7zip-full=
 p7zip-rar=
 rar=
-rpm=
+rpm=rpm2cpio
 rzip=
 tar=
 unace=
 unalz=
-unarchiver=
+unarchiver=unar
 unrar=
 unstaff=
 unzip=
-xz=
+xz=xz-utils
 zip=
 zoo=
 
